python/ray/serve/schema.py

```diff
@@ -597,24 +596,24 @@ class ServeDeploySchema(BaseModel):
           simply ignoring new parameters)
     """
 
-    proxy_location: DeploymentMode = Field(
-        default=DeploymentMode.EveryNode,
+    proxy_location: ProxyLocation = Field(
+        default=ProxyLocation.EveryNode,
         description=(
-            "The location of HTTP servers.\n"
-            '- "EveryNode" (default): start one HTTP server per node.\n'
-            '- "HeadOnly": start one HTTP server on the head node.\n'
-            '- "NoServer": disable HTTP server.'
+            "Config for where to run proxies for ingress traffic to the cluster.\n"
```

Comment:
```suggestion
            "Config for where to run proxies to receive ingress traffic to the cluster.\n"
```

Reply:
Also I think the formatting is a little borked... maybe try double newlines at the end?
<img width="707" alt="Screen Shot 2023-08-31 at 3 22 07 PM" src="https://github.com/ray-project/ray/assets/15851518/71fefbea-7bc6-4589-b694-7ef9641b1223">

Reply:
documenting the options is fully redundant because it links to the `ProxyLocation` ref, so just removing it here